layouts/page/nx-cugraph.html

```diff
@@ -0,0 +1,325 @@
+          
+          <br>
+          
+          In a simple script <code>demo.ipy</code>
+
+          <code class="code-sample mt-3">
+            import networkx as nx <br>
+            import pandas as pd <br>
+            <br>
+            url = "https://data.rapids.ai/cugraph/datasets/cit-Patents.csv" <br>
+            df = pd.read_csv(url, sep=" ", names=["src", "dst"], dtype="int32") <br>
+            G = nx.from_pandas_edgelist(df, source="src", target="dst") <br>
+            <br>
+            %time result = nx.betweenness_centrality(G, k=10)
+          </code>
+
+          <code class="code-sample mt-3">
+            user@machine:/# ipython demo.ipy<br>
+            <br>
+            CPU times: user 7min 36s, sys: 5.22 s, total: 7min 41s <br>
+            Wall time: 7min 41s <br>
+          </code>
+
+          <code class="code-sample mt-3">
+            user@machine:/# NX_CUGRAPH_AUTOCONFIG=True ipython demo.ipy<br>
+            <br>
+            CPU times: user 4.14 s, sys: 1.13 s, total: 5.27 s <br>
+            Wall time: 5.32 s<br>
+          </code>
+          <p class="fs-8 fw-lighter text-center">*NetworkX 3.4.1, nx-cugraph 24.10, CPU: Intel(R) Xeon(R) Gold 6128 CPU @ 3.40GHz 45GB RAM, GPU: NVIDIA Quadro RTX 8000 50GB RAM </p>
+        </div>
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-light fa-ballot-check"></i> Execution Flow </h2>
+          {{ with .Resources.GetMatch "nxcg-execution-diagram.jpg" }}
+          <img src="{{ .RelPermalink }}">
+          {{ end }}
+        </div>
+      </div>
+
+      <h1 class="mt-10 mb-4"> Try nx-cugraph Yourself </h1>
+      <p class="subheading  mb-5">Find where to get more resources and connect with developers.</p>
+
+      <div class="row">
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-light fa-bolt-lightning"></i> Try Now</h2>
+          <p>See how nx-cugraph can speedup your graph workflows in a free <a href="https://nvda.ws/4drM4re" target="_blank">GPU-enabled Colab notebook environment</a> using your Google account.</p>
+         
+          <br>
+         
+          <p>Additionally, nx-cugraph is now included in the RAPIDS metapackage. Visit the <a href="https://rapids.ai/#quick-start">RAPIDS Quick Start</a> to get started with RAPIDS on your favorite platform.</p>
+        </div>
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-light fa-comment-code"></i> Learn More </h2>
+          </p>Now nx-cugraph is generally available (GA) and ready for wide use. You can learn more from the <a href="https://developer.nvidia.com/blog/networkx-introduces-zero-code-change-acceleration-using-nvidia-cugraph/" target="_blank"> release blog</a> and <a href="https://networkx.org/documentation/stable/tutorial.html#using-networkx-backends" target="_blank"> nx-cugraph documentation</a>. </p>
```

Comment:
I expected the second link to go to the nx-cugraph docs, not the NetworkX docs on using backends.
```suggestion
          </p>Now nx-cugraph is generally available (GA) and ready for wide use. You can learn more from the <a href="https://developer.nvidia.com/blog/networkx-introduces-zero-code-change-acceleration-using-nvidia-cugraph/" target="_blank"> release blog</a> and <a href="https://docs.rapids.ai/api/cugraph/stable/nx_cugraph/nx_cugraph/" target="_blank"> nx-cugraph documentation</a>. </p>
```

Reply:
@nv-rliu is that correct?